.github/workflows/NPM-publish.yml

```diff
@@ -0,0 +1,48 @@
+# Trigger publish job on pushing release
+# The packages are published in the NPM registry.
+# There is a second job that could be enabled to publish into github NPM registry
+name: Release NPM package
+
+on:
+  release:
+    types:
+      - published
+
+
+jobs:
+  npm-registry-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
```

Comment:
BTW I don't see the node version requirement specified in the package.json on the other PR. You better add engines field and describe it in the readme as well. See here https://docs.npmjs.com/files/package.json#engines and here https://github.com/Ontotext-AD/graphdb.js/blob/master/package.json#L52

Reply:
Sure, I'll differently do that. Thanks for the heads up.